modules/gapi/test/common/gapi_imgproc_tests_inl.hpp

```diff
@@ -50,6 +50,27 @@ namespace
             rgb2yuyv(in_line_p, out_line_p, in.cols);
         }
     }
+
+    // Draw random ellipses on given mat of given size and type
+    void initMatForFindingContours(cv::Mat& mat, const cv::Size& sz, int type)
+    {
+        cv::RNG rng(time(nullptr));
```

Comment:
Never do that.

Tests must be reproducible (they are to catch regressions in the code, not a regressions from algorithms).
Reuse `cv::theRNG()` (probably by reference).

/cc @dmatveev

Reply:
Oh, I'm really sorry! I've got it now much more clearly, I think.
So, even if it is not what caused issues, I should get rid of this `time()` I've added to tests when will fix [this](https://github.com/opencv/opencv/issues/18373)...
Thank you!